src/sc/asyncproofverifier.cpp

```diff
@@ -72,6 +74,15 @@ void CScAsyncProofVerifier::RunPeriodicVerification()
      */
     uint32_t queueAge = 0;
 
+    int32_t delay = GetArg("-scproofverificationdelay", BATCH_VERIFICATION_MAX_DELAY);
+    assert(delay >= 0 && "scproofverificationdelay must be non negative");
+    uint32_t batchVerificationMaxDelay = static_cast<uint32_t>(delay);
+
+    int32_t size = GetArg("-scproofqueuesize", BATCH_VERIFICATION_MAX_SIZE);
+    assert(size >= 0 && "scproofqueuesize must be non negative");
+    uint32_t batchVerificationMaxSize = static_cast<uint32_t>(size);
+
+
```

Comment:
I wonder if we should restrict these two parameters to work in REGTEST mode only.

Reply:
I think they could be useful also for other network modes, and anyway a wrong usage should not harm other nodes. Probably we could add a range check for avoid absurd settings